### DISCUSSION TITLE
search results unfindable in hidden tabs

### DISCUSSION THREAD

**Discussion by CouldBeThis** [squidfunk/mkdocs-material#2580] [Q&A] [answered]
Hello! Thank you for the theme. :)

I am trying to solve a problem getting search to interact properly with [pymdownx.tabbed](https://facelessuser.github.io/pymdown-extensions/extensions/tabbed/). If you search for something, you go to the page the result is on, but the text is in any tab other than the foremost one, you will never find it. Ctrl-F cannot see it. 

Looked around material, mkdocs and pymdown repos to see if anyone else had mentioned this and was somewhat surprised to find absolutely nothing. 

By way of example, I was looking for how to change the password in DietPi, so I searched for "password" which brought me to [this page](https://dietpi.com/docs/dietpi_tools/), but the relevant info is obscured. (The answer is **DietPi Configuration > Security options**.)

So what is missing here? 

I did find this, [Interlinks between tabbed headers · Issue #947 · facelessuser/pymdown-extensions](https://github.com/facelessuser/pymdown-extensions/issues/947), in which linking to specific tabs is discussed in theory. Maybe that could be used to link directly to results in whichever tab they appear. Seems like a bit of a project though, and definitely beyond my capabilities. 

Would appreciate any thoughts. Thank you!!

**Comment by squidfunk** [comment, marked as answer]
> Ctrl-F cannot see it.

This is a limitation of the browser you're using (and probably all browsers), not Material for MkDocs.

**Reply by CouldBeThis** [reply]
Not wrong. Which is why I didn't open an issue. But I am totally stumped looking for a work around. 

What is everyone else doing? Very confused how it seems I am the only one bothered by this as it makes using the site extremely difficult. User error? Cranky user?

**Reply by RadheshRagunath** [reply]
I am also looking for a solution on this. Would be really great if any fix can be suggested for this!!

Thank you :-)
